src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -2580,19 +2580,19 @@ public static CameraData ToCameraData(this PerspectiveCamera camera, string name
         {
             var camData = new CameraData
             {
-                NearPlaneDistance = camera.NearPlaneDistance,
-                FarPlaneDistance = camera.FarPlaneDistance,
-
-                Name = name,
-                EyeX = camera.Position.X,
-                EyeY = camera.Position.Y,
-                EyeZ = camera.Position.Z,
-                LookX = camera.LookDirection.X,
-                LookY = camera.LookDirection.Y,
-                LookZ = camera.LookDirection.Z,
-                UpX = camera.UpDirection.X,
-                UpY = camera.UpDirection.Y,
-                UpZ = camera.UpDirection.Z
+                //NearPlaneDistance = camera.NearPlaneDistance,
```

Comment:
I'm guessing you're seeing a system invalid operation sometimes - please log a bug if you can repro this.

Reply:
Sure!
Sorry, I forgot to uncomment this.